src/ledger.ts

```diff
@@ -232,13 +234,17 @@ export class LedgerInteraction extends DirectKeystoreInteraction {
     return this.withTransport(async (transport) => {
       const response: AppAndVersion = await getAppAndVersion(transport);
       this.appVersion = response.version;
+      this.appName = response.name;
       return this.appVersion;
     });
   }
 
   async isLegacyApp(): Promise<boolean> {
     const version = await this.setAppVersion();
     const [majorVersion, minorVersion] = version.split(".");
+    // if the name includes "Legacy" then it is legacy app
+    if (this.appName && Boolean(this.appName.includes("Legacy"))) return false;
```

Comment:
should this return true?

Reply:
Ooof. Yes. Not enough coffee for me and not enough storage in my ledger to have both versions of the apps installed apparently

Reply:
Fixed. derp. Thanks for catching